mcs/class/System/Test/System.Net/HttpListener2Test.cs

```diff
@@ -154,7 +154,7 @@ public void Test1 ()
 			Send (ns, "GET / HTTP/1.1\r\n\r\n"); // No host
 			string response = Receive (ns, 512);
 			ns.Close ();
-			StringAssert.StartsWith ("HTTP/1.1 400", response);
+			Assert.IsTrue("HTTP/1.1 400".StartsWith (response));
```

Comment:
this is backwards, you need `response.StartsWith("HTTP/1.1 400")` instead, similar for all the changes in this file.